packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -869,14 +874,23 @@ export class MergeTree {
 			}
 			return true;
 		};
+
 		// Slide to the next farthest valid segment in the tree.
-		forwardExcursion(segment, goFurtherToFindSlideToSegment);
+		if (slidingPreference === SlidingPreference.Backward) {
```

Comment:
comment above and below (L888) should be updated, maybe just removed. i think it was less clear what the original version of the code did.